google/resource_google_folder_organization_policy_test.go

```diff
@@ -249,7 +283,7 @@ resource "google_folder" "orgpolicy" {
 }
 
 resource "google_folder_organization_policy" "bool" {
-	# Test numeric folder ID.
+    # Test numeric folder ID.
```

Comment:
nit: can you revert the extra indents here?

Reply:
done